eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -6,4 +6,7 @@
     <UsagePattern IdentityGlob="Microsoft.Internal.TestPlatform.Remote/17.6.33617.297" />
     <UsagePattern IdentityGlob="Microsoft.Extensions.DependencyModel/3.0.0-preview4-27615-11" />
   </IgnorePatterns>
+  <Usages>
```

Comment:
This should not be added to the prebuilt baseline.  This package version will need to be added to the source-build-reference-packages repo following the [Adding new Packages instructions](https://github.com/dotnet/source-build-reference-packages#adding-new-packages).  Once merged, this new version will need to flow into vstest.

Reply:
To generate a new package follow the instructions [here](https://github.com/dotnet/source-build-reference-packages#workflow).

FYI - Building is [only supported in a linux environment](https://github.com/dotnet/source-build-reference-packages#supported-platforms).

Reply:
Yeah I saw it's only on Linux and I am using subsystem and get the following errors:

![image](https://github.com/microsoft/vstest/assets/11340282/e17f0655-3ce0-4668-8985-b73e63ad8470)

I will test the generate.sh part

Reply:
It looks like your are running `./build.sh` this repo only supports building in the source-build configuration as noted in the instructions.  build with `./build.sh -sb`.  I apologize for the confusion.  We have an issue to address this.